ignition-server/controllers/local_ignitionprovider.go

```diff
@@ -87,6 +87,7 @@ var _ IgnitionProvider = (*LocalIgnitionProvider)(nil)
 
 const pullSecretName = "pull-secret"
 const additionalTrustBundleName = "user-ca-bundle"
+const managedTrustBundleName = "trusted-ca-bundle-managed"
```

Comment:
nit: probably should have a const () block now

Reply:
fixed